zuul.d/project-templates.yaml

```diff
@@ -663,7 +663,19 @@
         - ansible-test-sanity-docker-stable-2.11
         - ansible-test-units-kubernetes-core-python37
         - ansible-test-integration-kubernetes-core
-        - ansible-test-molecule-kubernetes-core
+        - ansible-test-molecule-splitter-kubernetes-core
+        - ansible-test-molecule-kubernetes-core_devel_0
+        - ansible-test-molecule-kubernetes-core_devel_1
+        - ansible-test-molecule-kubernetes-core_devel_2
+        - ansible-test-molecule-kubernetes-core_devel_3
+        - ansible-test-molecule-kubernetes-core_2.9_0
+        - ansible-test-molecule-kubernetes-core_2.9_1
+        - ansible-test-molecule-kubernetes-core_2.9_2
+        - ansible-test-molecule-kubernetes-core_2.9_3
+        - ansible-test-molecule-kubernetes-core_2.10_0
+        - ansible-test-molecule-kubernetes-core_2.10_1
+        - ansible-test-molecule-kubernetes-core_2.10_2
+        - ansible-test-molecule-kubernetes-core_2.10_3
```

Comment:
to be honest, I doubt you need to do this.  What we should do, is create a 2nd nodeset for k8s creating a controller / appliance model. Then, see how long the job runtimes are.

Reply:
We definitely need it, otherwise, the molecule testing is too long, I try using a controller and appliance and it does not reduce, the kubernetes collection is using the wait option in many test case, this increase the total elapsed time.

Reply:
what is too long, can you share job runtimes? Rather then dynamically splitting them, and maintaining that logic we can just make it static.